tests/behat/features/revoke.feature

```diff
@@ -6,16 +6,13 @@ Feature: See other devices and revoke their access
   So that I can revoke their access
 
   Background:
-    Given I am logged in with "ADMIN" permissions
```

Comment:
Worth having a separate test to ensure that ADMIN cannot revoke AUTHOR

OK to create a new issue for this instead of fixing on this PR

Reply:
Not sure about this one ... I think behat tests should mostly be aimed at testing the "happy path" and avoid testing edge cases.